package/gluon-respondd/src/respondd-nodeinfo.c

```diff
@@ -124,8 +128,10 @@ struct json_object * respondd_provider_nodeinfo(void) {
 
 	struct json_object *system = json_object_new_object();
 	json_object_object_add(system, "site_code", get_site_code());
-	if (gluonutil_has_domains())
+	if (gluonutil_has_domains()) {
 		json_object_object_add(system, "domain_code", get_domain_code());
+		json_object_object_add(system, "primary_domain_code", get_primary_domain_code());
```

Comment:
Hmm, `primary_domain_code` or `domain_code_primary`? I kinda like the latter, as the fields will be next to each other when sorted by keys.

Reply:
I would rather prefer the first. The second one might be confusing. It could also suggest, that there is a "primary" object with a special meaning or so...